plumbing/object/change.go

```diff
@@ -61,6 +70,12 @@ func (c *Change) Files() (from, to *File, err error) {
 	return
 }
 
+func isInvalidFileMode(t TreeEntry) bool {
```

Comment:
Please use the FileMode methods: https://godoc.org/gopkg.in/src-d/go-git.v4/plumbing/filemode

I think that is this method can be replaced with a call to `FileMode.IsRegular()`